site/docs/config.md

```diff
@@ -87,6 +87,8 @@ vega.expressionFunction('customFormatA', function(datum, params) {
 }
 ```
 
+<span class="vl-example" data-name="config_numberFormatType_tooltip"></span>
```

Comment:
Should this example be under this "### Providing Custom Formatters" section? 

I think it should has its own 
```
### Customize Formatter for Tooltips only

<describe that tooltip has more screen estate so you may want tooltip format to be a bit different than the default format.> For example, <describe the example a little bit more>.
```